test/dynamics/models/test_rotating_frame.py

```diff
@@ -534,75 +539,76 @@ def test_state_transformations_no_frame_Operator_types(self):
         rotating_frame = RotatingFrame(None)
 
         t = 0.123
-        y = Operator([1.0, 1j])
+        y = self.asarray(Operator([1.0, 1j]))
```

Comment:
I think this should remain as:

```suggestion
        y = Operator([1.0, 1j])
```
the point of the test is to make sure the `rotating_frame` functions behave properly if an `Operator` type is input. I think it's okay for this test to only use numpy.